app/src/main/res/layout/lr_phase_shift_dialog.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Yes. I have already created an issue #570  for taking care of this . I will add RC and LR phase shift experiments to this also. :+1: